builtin/logical/database/backend.go

```diff
@@ -174,6 +216,8 @@ func (b *databaseBackend) invalidate(ctx context.Context, key string) {
 	case strings.HasPrefix(key, databaseConfigPath):
 		name := strings.TrimPrefix(key, databaseConfigPath)
 		b.ClearConnection(name)
+	case strings.HasPrefix(key, databaseStaticRolePath):
```

Comment:
Does this case also need `b.ClearConnection(name)`?

Reply:
I don't believe so, but I'm not 100% sure. I'm not sure on what context `invalidate` is called, @briankassouf can you clarify?

Reply:
No, it doesn't. That's to invalidate connection parameters, roles are not cached in the same way

Reply:
Because of my other comment, i don't think we need to include this invalidation since no secondary should have a running queue.

Reply:
I removed the case statement in c9246ade3